graphstorm-processing/tests/test_dist_numerical_transformation.py

```diff
@@ -132,7 +171,7 @@ def test_numerical_transformation_with_median_imputer_and_std_norm(
     expected_imputed_std_ages = [0.2, 0.2, 0.1, 0.3, 0.2]
 
     for row, expected_val in zip(transformed_rows, expected_imputed_std_ages):
-        assert row["age"] == expected_val
+        assert_almost_equal(row["age"], expected_val, decimal=7)
```

Comment:
I guess this test was unstable because of float accuracy.

Reply:
Certainly. We may need to add decimal for every assert_almost_equal test on float number in the future.